std_msgs/Header header
float32 axis1
float32 axis2
float32 axis3
float32 axis4
float32 axis5
float32 axis6
float32 gripper